orion/pkg/domains/achieve.go

```diff
@@ -18,6 +18,11 @@ type Achieve struct {
 	Message   string       `json:"message"`
 }
 
+type AchieveList struct {
```

Comment:
anyway we can rename this to represent some sort of grouping with years? Like `AchieveYearList` or `AchieveYearGroup` or `AchieveByYear`?

Reply:
The reason why I'm whining about this is because `AchieveList` is a bit too general and doesn't really convey what this struct represents.

Imagine a new intern coming (which may very well soon come) and looking at this struct, they'd probably have to guess what this does or ask the originator (you) what this does. And if this cycle continues, in which a lot of interns come and go, you can probably guess how quickly this codebase can erode.